paper/paper.md

```diff
@@ -38,9 +38,33 @@ TO DO
 
 TO DO
 
-# Mathematics
+# HyP3 Description
+
+## Development History
+HyP3 grew out of a need for a standardized pipeline to perform image processing workflows at the Alaska Satellite Facility (ASF) Distributed Active Archive Center (DAAC). HyP3 was initially created in 2016 by a group of undergraduate students interning at ASF. These students created a prototype version of HyP3 that was designed to handle workflows running on-premises and in the AWS cloud. This prototype was iterated upon for several more years, and by 2019 it was capable of performing many SAR-specific image processing workflows.
+
+By 2019 it was clear that re-designing HyP3 to take advantage of evolving technologies (namely the new AWS Batch service, infrastructure-as-code, and containerized workflows) would greatly improve its utility. Thus, in 2019 HyP3 was re-designed to run entirely within the AWS cloud, with application-specific processing workflows encapsulated within Docker containers. The re-design was completed in 2020, and on-demand SAR image processing services that utilized HyP3 were made available to the public at no cost via an API, Python SDK, and ASF's Data Search Vertex website.
+
+The first re-designed on-demand workflow made publicly available was Radiometric Terrain Correction (RTC) in [month year], followed by InSAR in [month year]. Since that time, new image processing services have been added to HyP3 including a glacier velocity tracking tool, and flood inundation mapping workflows. The user base grew to XX by [date], with users producing XX image products on average every month. Since 2021, a range of users have recognized the utility of HyP3. HyP3 was deployed within NASA's Earthdata Cloud in 2022, and many other deployments of HyP3 now exist at other institutions (see the Ongoing Projects section for details).
+
+## HyP3 Architecture
+HyP3 is a cloud processing pipeline that is designed with a serverless architecture and follows the AWS Well-Architected Framework. It also strives to make a clear separation between the plugins that perform the image processing and the pipeline in which this task is performed. See Figure FF for the AWS architecture diagram.
```

Comment:
I've never heard of the AWS Well-Architected Framework so I doubt we intentionally followed it for the design of HyP3, but happy to be corrected.
```suggestion
HyP3 is a cloud processing pipeline with a serverless architecture. The design makes a clear distinction between HyP3 itself and the image processing plugins. See Figure FF for the AWS architecture diagram.
```

Reply:
Good point, Jake. I put this the AWS Framework here because it seemed more specific than "best practices," but maybe just removing that comment would be the best move. We can talk about specific best practices we emphasized if we'd like, but otherwise it's a big vague.

Reply:
You hear about the well-architected framework A LOT when you take AWS certification trainings. I've reached out to Andrew on MM to get his take.